tests/impl.cpp

```diff
@@ -9149,14 +9149,287 @@ result_t test_mm_cmpestrc(const SSE2NEONTestImpl &impl, uint32_t iter)
+            val[8], val[9], val[10], val[11], val[12], val[13], val[14],      \
+            val[15]);                                                         \
+        if (memcmp(expect, &res, sizeof(__m128i))) {                          \
+            uint8_t val[16];                                                  \
+            memcpy(val, &res, sizeof(val));                                   \
+            printf("ubyte %d ", c);                                           \
+            printf(                                                           \
+                "Numerical: %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d " \
+                "\n",                                                         \
+                val[0], val[1], val[2], val[3], val[4], val[5], val[6],       \
+                val[7], val[8], val[9], val[10], val[11], val[12], val[13],   \
+                val[14], val[15]);                                            \
+            return TEST_FAIL;                                                 \
+        }                                                                     \
+    } while (0);
+
+#define EVAL_MM_CMPESTRM_SBYTE_TEST_CASE(c)                                   \
+    do {                                                                      \
+        int8_t *a = test_mm_cmpestrm_sbyte_data[c].a,                         \
+               *b = test_mm_cmpestrm_sbyte_data[c].b;                         \
+        int la = test_mm_cmpestrm_sbyte_data[c].la,                           \
+            lb = test_mm_cmpestrm_sbyte_data[c].lb;                           \
+        const int imm8 = IMM_##c;                                             \
+        int8_t *expect = test_mm_cmpestrm_sbyte_data[c].expect;               \
+        __m128i ma, mb, res;                                                  \
+        memcpy(&ma, a, sizeof(ma));                                           \
+        memcpy(&mb, b, sizeof(mb));                                           \
+        res = _mm_cmpestrm(ma, la, mb, lb, imm8);                             \
+        int8_t val[16];                                                       \
+        memcpy(val, &ma, sizeof(val));                                        \
+        printf("sbyte a %d ", c);                                             \
+        printf(                                                               \
+            "Numerical: %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d \n",  \
+            val[0], val[1], val[2], val[3], val[4], val[5], val[6], val[7],   \
+            val[8], val[9], val[10], val[11], val[12], val[13], val[14],      \
+            val[15]);                                                         \
+        memcpy(val, &mb, sizeof(val));                                        \
+        printf("sbyte b %d ", c);                                             \
```

Comment:
Is `printf` necessary?

Reply:
> Is `printf` necessary?

This `printf` is used for solely debug purpose. I will remove it as soon as I finish the test cases verification.